modules/external-auth-disabling.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-auth-disabling_{context}"]
+= Disabling direct authentication
+
+If necessary, you can disable direct authentication for your cluster and revert back to authenticating with the built-in OpenShift OAuth server.
+
+.Prerequisites
+
+* You have access to the `kubeconfig` file for the cluster.
+
+.Procedure
+
+. Log in to the cluster by using your `kubeconfig` file.
++
+You must use the `kubeconfig` file to configure the direct authentication. If you use a token, the token can become invalid during the process of the procedure.
```

Comment:
@bergerhoffer , as https://github.com/openshift/openshift-docs/pull/93779/files#r2126406908 , customers can use a long-lived serviceaccount token kubeconfig file as long as that serviceaccount is granted the needed admin perm. If that comment explains well what a valid kubeconfig file is, you can remove "If you use a token, the token can become invalid during the process of the procedure" (here and another occurrence)